org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -37,6 +37,9 @@
   <li>Exclude from a report methods which are annotated with <code>@lombok.Generated</code>.
       Initial analysis and contribution by Rüdiger zu Dohna.
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/513">#513</a>).</li>
+  <li>Exclude from a report methods which are annotated with <code>@groovy.transform.Generated</code>
+      to better integrate with Groovy >= 2.6.0. Thanks to Andres Almiray to make this possible!
```

Comment:
I'm not a native English speaker, but "**to** make this possible" looks/sounds weird for me.

Reply:
Also I believe that `>= 2.6.0` should be replaced by `>= 2.5.0` since change on Groovy side was merged not just into `master` and `2_6_x` branches, but also into `2_5_x`.

Reply:
I guess "Thanks to Andres Almiray **for making** this possible!" would work, but perhaps just rephrase to match the preceding bullet or say "for the contribution" or something.

Reply:
@bjkail Thanks for your input! What I want to express here that Andres triggered the introduction of the annotation in the Groovy compiler.

Reply:
Ah, then "for making this possible" would work, but "this" seems ambiguous, so I would say "for adding the annotation to Groovy, which made this possible".

Reply:
@Godin @bjkail Fixed wording and version range :white_check_mark: